ghost/core/test/unit/server/services/explore-ping/ExplorePingService.test.js

```diff
@@ -23,6 +23,8 @@ describe('ExplorePingService', function () {
         settingsCacheStub.get.withArgs('active_theme').returns('alto');
         settingsCacheStub.get.withArgs('explore_ping').returns(true);
         settingsCacheStub.get.withArgs('explore_ping_growth').returns(false);
+        settingsCacheStub.get.withArgs('facebook').returns('https://facebook.com/example');
+        settingsCacheStub.get.withArgs('twitter').returns('https://twitter.com/example');
```

Comment:
In settings table, we don't store the entire Twitter/Facebook URLs, only the usernames:
- Facebook: `example`
- Twitter: `@example`

So think we either want to:
1. Update the tests to reflect that we're returning usernames only, not full URLs
2. Or, transform the usernames into full URLs before calling the Explore service

Reply:
Have changed the tests.